### ISSUE TITLE
Fix thumbnail aspect ratios

### ISSUE THREAD

**Issue by Samq64** [ScratchAddons/ScratchAddons#3944]
### Is your request related to a problem? Please describe.

Scratch displays thumbnails at inconsistent aspect ratios throughout the site causing some thumbnails (especially studios) to appear squashed (or stretched, but I haven't seen that yet)

### Describe the solution you'd like

An addon that makes all profile pictures 1:1, all project thumbnails 4:3 and all studio thumbnails 16:9.

### Describe alternatives you've considered

Making it an option for `hi-res-thumbnails`. That might actually work better since that addon is already going though all the thumbnails on the site.

**Comment by Samq64** [comment]
Maybe this addon could also give the option whither thumbnails should be cropped or stretched if they were uploaded at a strange aspect ratio. (profile pictures and studios, also occasionally happens with projects)

**Comment by mxmou** [comment]
[Related](https://github.com/LLK/scratch-www/issues/5598#issuecomment-864118903)

Project thumbnails use the correct aspect ratio everywhere except in My Stuff, but studio thumbnails use three different aspect ratios: 20 : 13 on the studio page, 17 : 10 on most other pages and 5 : 4 in My Stuff. The most common one, 17 : 10, should probably be used.

> Maybe this addon could also have an option to make thumbnails cropped intead of stretched.

That would make some thumbnails better and some worse. I don't think the resizing behavior of project or studio thumbnails should be changed, but an option to choose the preferred behavior of profile pictures would actually be useful: Scratch stretches them in most places, but not on project and curator tiles on the studio page.

**Comment by Secret-chest** [comment]
> 16:9

As @mxmou said, they are not 16:9.

**Comment by Samq64** [comment]
> As @mxmou said, they are not 16:9.

I didn't bother editing it. Also, I have no idea why studio thumbnails are not a standard aspect ratio. 17:10 is very close to 16:9 and I'm pretty sure most people who care about good thumbnails think "This looks like it's 16:9, so I'll make my image 16:9 before uploading it." (At least I do)

**Comment by towerofnix** [comment]
> Making it an option for `hi-res-thumbnails`. That might actually work better since that addon is already going though all the thumbnails on the site.

Agreed here! `hi-res-thumbnails` operates on the element size of thumbnails (for both downscaling and upscaling), so it's important that any of its behavior happen *after* behavior which alters the size of said thumbnails. Since we're already operating on dimensions and the like, it makes sense to do that as part of the same addon, rather than having two separate, closely related addons (which a thumbnail-attentive user will probably use together anyway).
